helm_deploy/values-dev.yaml

```diff
@@ -26,14 +25,6 @@ env:
   WIND_BACK_SECONDS: "10"
   APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
 
-whitelist:
```

Comment:
What are these changes for?

Reply:
These are to match the other repos. To not limit access to office/vpn etc.